src/main/client/src/components/app/table/react-table/ReactTableView.js

```diff
@@ -55,14 +57,16 @@ class ReactTableView extends Component<Props, State> {
         const layerListChanged = JSON.stringify(nextProps.layerList) !== JSON.stringify(layerList);
         const filteredColumnsChanged = nextProps.columns.length
             !== columns.length;
+        const pageChanged = nextProps.activePage;
```

Comment:
This should compare value to activePage as follows:
const pageChanged = nextProps.activePage !== activePage;